src/lib/tenant/tenantContractFactory.ts

```diff
@@ -34,6 +35,39 @@ export default class TenantContractFactory {
   }
 }
 
+const ensContracts = (isProd: boolean): TenantContracts => {
+  return {
+    // TOKEN
+    token: new TenantContract<ITokenContract>({
+      contract: EtherfiToken__factory.connect(
+        "0xC18360217D8F7Ab5e7c516566761Ea12Ce7F9D72",
+        ethProvider
+      ),
+      address: "0xC18360217D8F7Ab5e7c516566761Ea12Ce7F9D72" as `0x${string}`,
+      chainId: 1,
+      chainName: "Ethereum Mainnet",
+      abi: EtherfiToken__factory.abi,
```

Comment:
Is this supposed to be `ENSToken_factory`?  or it's generic ERC-20 contract?

Reply:
@stepandel, that's a temporary placeholder. For ENS, Etherfi, and all other ERC20 tokens, I plan to create an interchangeable class. This works for now but I will make a change this week.